### ISSUE TITLE
Audio Streaming broken

### ISSUE THREAD

**Issue by mahnerak** [gradio-app/gradio#1332]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

It seems the new Audio component does not support audio streaming in live mode. The stream is sent only after you click stop.
But it does work on `gradio==2.9`.

### Reproduction

Install `gradio>3.0`.
Run any realtime audio app (say, https://huggingface.co/spaces/YSU/aspram-realtime/blob/main/app.py). It will send the data after you click stop instead of continuously sending chunks.

### System Info

```shell
gradio==3.0
```

### Severity

critical

**Comment by tango4j** [comment]
I am having the same issue (audio streaming is not activated when it is running on local machine). Also, gradio.inputs.Audio() does not have argument "streaming" unlike described in the docs.

**Comment by abidlabs** [comment]
Thanks @mahnerak and @tango4j for pointing this out! We are looking into this

**Comment by aliabid94** [comment]
ahh I see, can you guys try using `gr.Audio(source="microphone", streaming=True)` (not `gr.inputs.Audio`) for the input, and let me know if that works? Will figure out the right way to bring it back to the old `gr.inputs.Audio` in the meantime.

**Comment by mahnerak** [comment]
The problem is there even when you're not using streaming=True.
You can use just live mode on which will rerun gradio pipeline every T secs.

Starting 3.0 you should click stop to make gradio send the request to the backend:

https://user-images.githubusercontent.com/1367529/169415102-cc1200b2-d742-43f1-b0cf-cb968957c4b8.mov


In 2.9 it works as expected. You can note pulsating icon of processing while I'm still recording the audio:

https://user-images.githubusercontent.com/1367529/169415125-446b2bb6-781b-4bf0-b9c4-4b9660f57446.mov

BTW, You can access exactly the same gradio app here:
https://huggingface.co/spaces/YSU/aspram-realtime
I've changed gradio version to 2.9 (in README, yaml on the top) to make it work.


Hope this helps.

**Comment by aliabid94** [comment]
Fixing now, thanks for the thorough documentation!

**Comment by abidlabs** [comment]
Hi @mahnerak and @tango4j, this should be fixed now! We'll release `gradio==3.1` soon, and you can try the following example as a simple streaming speech to text example:

```py
from transformers import pipeline
import gradio as gr
import time

p = pipeline("automatic-speech-recognition")

def transcribe(audio, state=""):
    time.sleep(2)
    text = p(audio)["text"]
    state += text + " "
    return state, state

gr.Interface(
    fn=transcribe, 
    inputs=[
        gr.Audio(source="microphone", type="filepath", streaming=True), 
        "state"
    ],
    outputs=[
        "textbox",
        "state"
    ],
    live=True).launch()
```

**Comment by rakesh-clinicyantra** [comment]
> ```python
> from transformers import pipeline
> import gradio as gr
> import time
> 
> p = pipeline("automatic-speech-recognition")
> 
> def transcribe(audio, state=""):
>     time.sleep(2)
>     text = p(audio)["text"]
>     state += text + " "
>     return state, state
> 
> gr.Interface(
>     fn=transcribe, 
>     inputs=[
>         gr.Audio(source="microphone", type="filepath", streaming=True), 
>         "state"
>     ],
>     outputs=[
>         "textbox",
>         "state"
>     ],
>     live=True).launch()
> ```

Thanks!!
